.env.example

```diff
@@ -0,0 +1,4 @@
+DATABASE_URL=postgresql://postgres:DCpsafNCdC4lrAX5jNI0@containers-us-west-111.railway.app:7204/railway
+GOOGLE_CLIENT_ID=936456571444-qsvgo4fr8qe4r0vmkciqk31f9b9p1cqj.apps.googleusercontent.com
+GOOGLE_CLIENT_SECRET=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
+GOOGLE_SECRET=H62sqW9KG821i9LqnMGkA+356x2pIdoQ+OtZ+86qG7Q=
```

Comment:
gut guard won't pass, so just make them empty for now. I'll ask to share it later.

```
DATABASE_URL=
GOOGLE_CLIENT_ID=
GOOGLE_CLIENT_SECRET=
GOOGLE_SECRET=
```